actions/serviceacts.go

```diff
@@ -591,6 +595,11 @@ func printNoteAndSols(writer io.Writer, tuneApp *app.App, jstat *system.JStatus)
 	appliedNotes := tuneApp.AppliedNotes()
 	fmt.Fprintf(writer, "%s", appliedNotes)
 	fmt.Fprintf(writer, "\n")
+	fmt.Fprintf(writer, "orphaned Overrides:       ")
```

Comment:
why this many spaces?

Reply:
eye-catching and readability of the output of 'saptune status'. Having all values of the status output starting at the same position in the row, having the impression of a table.

Reply:
Awesome, ok!